generators/base-application/tasks.d.mts

```diff
@@ -1,31 +1,36 @@
-import {
-  GenerericTaskParam,
-  GenericTaskGroup,
-  InitializingTaskGroup as BaseInitializingTaskGroup,
-  PromptingTaskGroup as BasePromptingTaskGroup,
-  ConfiguringTaskGroup as BaseConfiguringTaskGroup,
-  ComposingTaskGroup as BaseComposingTaskGroup,
-  LoadingTaskGroup as BaseLoadingTaskGroup,
-} from '../base/tasks.mjs';
+import { ControlTaskParam, BaseGeneratorDefinition, SourceTaskParam } from '../base/tasks.mjs';
+import { BaseApplication } from './types.mjs';
 
-export type Application = Record<string, any>;
+type Field = {
+  fieldName: string;
+  fieldType: string;
+  fieldTypeBlobContent: string;
```

Comment:
Usage of decorator pattern? BlobType is not always filled

Reply:
This was moved from below.
Fields types will be worked on later